src/plugins/gs-plugin-external-apps.c

```diff
@@ -43,8 +44,15 @@
 
 #define TMP_ASSSETS_PREFIX "gs-external-apps"
 
+enum GsPluginExternalType {
+	GS_PLUGIN_EXTERNAL_TYPE_DEB = 1,
+	GS_PLUGIN_EXTERNAL_TYPE_TAR,
+	GS_PLUGIN_EXTERNAL_TYPE_UNKNOWN,
```

Comment:
Extra comma here?
Also, I wonder whether the TYPE_UNKNOWN should come as the first type, so it's value doesn't keep changing (assuming you'd keep UNKNOWN as the last type).